### PR TITLE
added style and form on vinotheque page, update background

### PR DESCRIPTION
**Pull request by MarjorieThomassin**

### CODE REVIEW

File: src/pages/Home.css [hunk 1]
```diff
@@ -76,6 +83,7 @@ nav {
     display: flex;
     align-items: center;
     padding: 0.5rem;
+    margin-left: 10vh;
```
Comment (rocambille) [inline src/pages/Home.css:86]: you should use no more than 4rem for margin: if you need more, you're facing a layout problem, not spacing. try flexbox/grid on the container instead

File: src/pages/Profile.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
+.formContainer {
+    border: solid #c49e34;
+    border-radius: 5%;
+    width: 50%;
+    padding: 0.8rem;
+    margin: auto;
+    margin-top: 2%;
```
Comment (rocambille) [inline src/pages/Profile.css:7]: you should use rem for margins, and no more than 4rem for margin: if you need more, you're facing a layout problem, not spacing. try flexbox/grid on the container instead

File: src/pages/Profile.css [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+.inputBottle {
+    font-weight: bold;
+    font-size:1.5rem;
+    padding: 0.25rem;
+    border: 1px solid black;
+    background-color: white;
+    border-radius: 5px;
+    margin: 10px;
```
Comment (rocambille) [inline src/pages/Profile.css:27]: you should use rem for margins, and no more than 4rem for margin: if you need more, you're facing a layout problem, not spacing. try flexbox/grid on the container instead

**Review by rocambille — CHANGES_REQUESTED** [review]

File: src/pages/Profile.css (hunk 3 above)
Comment (MarjorieThomassin) [reply src/pages/Profile.css]: Fixed

File: src/pages/Home.css (hunk 1 above)
Comment (MarjorieThomassin) [reply src/pages/Home.css]: Fixed

**Pull request merged**